pkg/cmd/test.go

```diff
@@ -75,63 +85,89 @@ type testCmdOptions struct {
 	env          []string
 }
 
+func (o *testCmdOptions) validateArgs(_ *cobra.Command, args []string) error {
+	if len(args) != 1 {
+		return errors.New(fmt.Sprintf("accepts exactly 1 test name to execute, received %d", len(args)))
+	}
+
+	return nil
+}
+
 func (o *testCmdOptions) run(_ *cobra.Command, args []string) error {
+	var err error
+	source := args[0]
```

Comment:
I think this no longer works with multiple test files, and panics with no test files passed.

Reply:
We should add some tests to avoid breaking things